code/modules/mob/living/carbon/xenomorph/abilities/lurker/lurker_powers.dm

```diff
@@ -282,18 +282,9 @@
 
 	stab_direction = turn(get_dir(xeno, targeted_atom), 180)
 	playsound(hit_target,'sound/weapons/alien_tail_attack.ogg', 50, TRUE)
+	if(hit_target.mob_size < MOB_SIZE_BIG)
+		step_away(hit_target, xeno)
 
-	var/direction = Get_Compass_Dir(xeno, targeted_atom) //More precise than get_dir.
-
-	if(!step(hit_target, direction))
```

Comment:
We should only enter this block if `hit_target.mob_size < MOB_SIZE_BIG` and if not just slow them. (meaning the else slow should be tabbed out 1 level)